build.gradle

```diff
@@ -31,11 +31,13 @@ dependencies {
     compile group: "com.fasterxml.jackson.core", name: "jackson-databind", version: "2.5.0"
     compile group: "joda-time", name: "joda-time", version: "2.3"
     compile group: "com.fasterxml.jackson.datatype", name: "jackson-datatype-joda", version: "2.0.4"
+    compile 'org.slf4j:slf4j-api:1.7.12'
```

Comment:
Lets keep the convention. If we use map syntax (group:, name:, etc) lets keep it consistent.